### PR TITLE
Updated caption in md file

### PR DESCRIPTION
**Pull request by marksguitars**
Please be nice and approve this pull request :)

Keywords:

- Closes
- Fixes
- Resolves

This PR resolves #2

### CODE REVIEW

**Review by kennethhowe — APPROVED** [review]

File: _slides/13-marksguitars.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hi Erica!
```
Comment (kennethhowe) [inline _slides/13-marksguitars.md]: 👍

Comment (kennethhowe) [inline _slides/13-marksguitars.md]: This is another one to make it like the class

Comment (kennethhowe) [inline _slides/13-marksguitars.md]: Trying a suggest
```suggestion
Hi Erica!!!!!
```

**Review by kennethhowe — APPROVED** [review]

**Comment by marksguitars** [comment]
Merging back into main.

**Pull request merged**